asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

```diff
@@ -354,7 +354,7 @@ NOTE: <<ref_tbl_pid10_mapping>> defines the mapping of the SDC patient's race in
 |Alternate Coding System Version ID
 |pm:PatientContextState+++<wbr/>+++/pm:CoreData+++<wbr/>+++/pm:Race+++<wbr/>+++/pm:Translation
 /@CodingSystemVersion
-|Note that only the first entry of the *pm:Translation* element list is to be mapped.
+|Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. In this case, only the first entry of the *pm:Translation* element list is to be mapped.

```

Comment:
```suggestion
|Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. Otherwise only the first entry of the *pm:Translation* element list is to be mapped.

```